indigo/distribution.yaml

```diff
@@ -3533,6 +3533,18 @@ repositories:
       url: https://github.com/ual-arm-ros-pkg/mvsim.git
       version: master
     status: maintained
+  myo_ros:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/clearpath-gbp/myo_ros-release.git
+      version: 0.1.2-0
+    source:
+      type: git
+      url: https://github.com/clearpathrobotics/myo_ros.git
+      version: master
+    status: maintained
+    status_description: New release. Adding examples planned, but low-priority.
```

Comment:
Please remove the status description since it is not intended for such information.